File: edabitPython/sum_missing_numbers.py
# https://edabit.com/challenge/e3KZvJbSXeHXtJA7c
def sum_missing_numbers(lst):
	sum = 0
	for i in range(min(lst), max(lst)):
		if lst.count(i) == 0:
			sum += i
	return sum
	

File: edabitJavascript/sumMissingNumbers.js
function sumMissingNumbers(arr) {
  let min = Math.min(...arr);
  let max = Math.max(...arr);
  let sum = 0;
  for (let i = min; i < max; i++) {
    if (!arr.includes(i)) {
      sum += i;
    }
  }
  return sum;
}
